soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -13218,17 +13229,16 @@ void func_8084FB10(Player* this, GlobalContext* globalCtx) {
 }
 
 void func_8084FBF4(Player* this, GlobalContext* globalCtx) {
-    LinkAnimation_Update(globalCtx, &this->skelAnime);
-    func_808382BC(this);
+        LinkAnimation_Update(globalCtx, &this->skelAnime);
+        func_808382BC(this);
 
-    if (((this->unk_850 % 25) != 0) || func_80837B18(globalCtx, this, -1)) {
-        if (DECR(this->unk_850) == 0) {
-            func_80839F90(this, globalCtx);
+        if (((this->unk_850 % 25) != 0) || func_80837B18(globalCtx, this, -1)) {
+            if (DECR(this->unk_850) == 0) {
+                func_80839F90(this, globalCtx);
+            }
         }
-    }
-
-    this->shockTimer = 40;
-    func_8002F8F0(&this->actor, NA_SE_VO_LI_TAKEN_AWAY - SFX_FLAG + this->ageProperties->unk_92);
+        this->shockTimer = 40;
+        func_8002F8F0(&this->actor, NA_SE_VO_LI_TAKEN_AWAY - SFX_FLAG + this->ageProperties->unk_92);
 }
```

Comment:
again, was any of this modified functionality? doesn't appear to be, if not, revert that part.